src/assets/news-api-dummy.ts

```diff
@@ -0,0 +1,89 @@
+import {INewsFlash} from "../models/NewFlash";
+
+export const newsApiDummy :Array<INewsFlash>= [
+  {
+    "id": 13539,
+    "lat": 32.01857,
+    "lon": 34.906059,
+    "title": "\u05d4\u05d5\u05ea\u05e8 \u05dc\u05e4\u05e8\u05e1\u05d5\u05dd: \u05d9\u05d5\u05e1\u05e3 \u05e7\u05d4\u05dc\u05e0\u05d9, \u05d1\u05df 79 \u05de\u05e4\u05ea\u05d7 \u05ea\u05e7\u05d5\u05d5\u05d4, \u05d4\u05d5\u05d0 \u05d0\u05d7\u05d3 \u05d4\u05d4\u05e8\u05d5\u05d2\u05d9\u05dd \u05d1\u05ea\u05d0\u05d5\u05e0\u05d4 \u05d4\u05e7\u05d8\u05dc\u05e0\u05d9\u05ea",
+    "source": "ynet",
+    "date": new Date("2019-12-23 12:33:54")
+  },
+  {
+    "id": 13530,
+    "lat": 32.01857,
+    "lon": 34.906059,
+    "title": "\u05d1\u05ea\u05d5 \u05e9\u05dc \u05d4\u05d4\u05e8\u05d5\u05d2 \u05d1\u05ea\u05d0\u05d5\u05e0\u05d4 \u05d4\u05e7\u05d8\u05dc\u05e0\u05d9\u05ea: \"\u05d0\u05d9\u05df \u05d1\u05d9 \u05db\u05e2\u05e1, \u05d6\u05d4 \u05dc\u05d0 \u05d9\u05d7\u05d6\u05d9\u05e8 \u05d0\u05ea \u05d0\u05d1\u05d0 \u05e9\u05dc\u05d9\"",
+    "source": "ynet",
+    "date": new Date("2019-12-23 10:35:25")
```

Comment:
We'll probably receive the data as date strings, not as Date objects.
Please verify which one happens
- Some middleware/parser/something converts the received data to date objects (so this is okay)
- We have to deal with strings from the data (so this is not okay, and you'll have to remove `new Date(...)` and leave the strings.